docs/ops/activation/Clamp_1.md

```diff
@@ -6,48 +6,63 @@
 
 **Short description**: *Clamp* operation represents clipping activation function.
 
+**Detailed description**:
+
+*Clamp* performs clipping operation over the input tensor element-wise. Element values of the output are within the range `[min, max]`.
+* Input values that are smaller than *min* are replaced with *min* value. For example, *min* equal 10 means that any value in the input that is smaller than the bound, is replaced by 10.
+* Input values that are greater than *max* are replaced with the *max* value. For example, *max* equals 50 means that any value in the input that is greater than the bound, is replaced by 50.
```

Comment:
I think that the explanation is clear enough and sentences "For example, *min* equal 10 means that any value in the input that is smaller than the bound, is replaced by 10." can be removed. The same for max

Reply:
I agree. Removed